.docker/scripts/init.sh

```diff
@@ -9,6 +9,7 @@ if [ "$VENV_DIR" != "" ]; then
 fi
 CLICK_ARGS="--yes" ckan_cli db clean
 ckan_cli db init
+ckan_cli db upgrade
```

Comment:
So init is not enough?

Reply:
It should be enough, this is just syncing with another repo.